src/Rules/Library/ChromiumComponentsShouldUseWebScanner.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Axe.Windows.Rules.Resources;
+using static Axe.Windows.Rules.PropertyConditions.Framework;
+
+namespace Axe.Windows.Rules.Library
+{
+    [RuleInfo(ID = RuleId.ChromiumComponentsShouldUseWebScanner)]
+    class ChromiumComponentsShouldUseWebScanner : Rule
+    {
+        public ChromiumComponentsShouldUseWebScanner()
+        {
+            Info.Description = Descriptions.ChromiumComponentsShouldUseWebScanner;
+            Info.HowToFix = HowToFix.ChromiumComponentsShouldUseWebScanner;
+            Info.ErrorCode = EvaluationCode.Error;
```

Comment:
At one point, someone mentioned treating this as a framework issue, which would potentially mean creating a web page and linking to it. Did we ultimately decide against that approach?

Reply:
I wasn't aware of that discussion and Fer is OOF so we won't be able to get her input for a while. Is that something that had been settled previously?